packages/pn-personagiuridica-webapp/src/navigation/__test__/SessionGuard.test.tsx

```diff
@@ -23,6 +23,7 @@ vi.mock('react-router-dom', async () => ({
 vi.mock('react-i18next', () => ({
   useTranslation: () => ({
     t: (str: string) => str,
+    i18n: { changeLanguage: () => new Promise(() => {}) },
```

Comment:
Remove this

Reply:
If I remove this lines the test throws an error:
`TypeError: Cannot read properties of undefined (reading 'changeLanguage')`

Reply:
Moving `handleSetUserLanguage` to `App.tsx` resolved the error.